stance/predict/overlay.py

```diff
@@ -13,7 +16,51 @@ def display_overlay(input_video, template_skeletons):
         path to the video of the user performing the motion
     template_skeletons : List[Skeletons]
     """
-    pass
+    cap = cv2.VideoCapture(input_video)
+    numFrames = int(cap.get(cv2.CAP_PROP_FRAME_COUNT))
+    width = int(cap.get(cv2.CAP_PROP_FRAME_WIDTH))
+    height = int(cap.get(cv2.CAP_PROP_FRAME_HEIGHT))
+
+    # used for scaling template skeleton to user video
+    scale_x = width / 1280
```

Comment:
Where are 1280 and 720 coming from?

Reply:
It looks like you've hard-coded it for the squat.mp4, when I run your code with the other two examples, it either shows it too far to the right, or not at all.